lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -148,7 +148,7 @@ def _parse_proxy_command(self, port=22):
             getattr(self._play_context, 'ssh_args', ''),
         ]
         if ssh_args is not None:
-            args = self._split_ssh_args(' '.join(ssh_args))
+            args = self._split_ssh_args(' '.join(filter(None,ssh_args)))
```

Comment:
Note -- rather than using filter, it's more pythonic to use a generator expression or list comprehension like this:

``` python
args = self._split_ssh_args(' '.join(a for a in ssh_args if a is not None))
```

It can also be quicker if you are dealing with a large list (not the case here).